source/Calamari.Tests/AWS/CloudFormation/CloudFormationFixture.cs

```diff
@@ -41,157 +17,37 @@ public class CloudFormationFixture
         [Category(TestCategory.CompatibleOS.OnlyWindows)]
         public async Task DeleteCloudFormationStack()
         {
-            var template = $@"
-{{
-    ""Resources"": {{
-        ""{StackName}"": {{
-            ""Type"": ""AWS::S3::Bucket"",
-            ""Properties"": {{ 
-                ""BucketName"": {StackName}
-            }}
-        }}
-    }}
-}}";
-            DeployTemplate(StackName, template);
-            DeleteStack(StackName);
-            await ValidateStackExists(StackName, false);
-        }
-
-        void ValidateS3(Action<AmazonS3Client> execute)
-        {
-            var credentials = new BasicAWSCredentials(
-                Environment.GetEnvironmentVariable("AWS_Calamari_Access"),
-                Environment.GetEnvironmentVariable("AWS_Calamari_Secret"));
-            var config = new AmazonS3Config { AllowAutoRedirect = true, RegionEndpoint = RegionEndpoint.APSoutheast1 };
-            using (var client = new AmazonS3Client(credentials, config))
-            {
-                execute(client);
-            }
-        }
-
-        async Task ValidateStackExists(string stackName, bool shouldExist)
-        {
-            await ValidateCloudFormation(async (client) =>
-            {
-                Func<IAmazonCloudFormation> clientFactory = () => client;
-                var stackStatus = await clientFactory.StackExistsAsync(new StackArn(stackName), Aws.Deployment.Conventions.StackStatus.DoesNotExist);
-                stackStatus.Should().Be(shouldExist ? Aws.Deployment.Conventions.StackStatus.Completed : Aws.Deployment.Conventions.StackStatus.DoesNotExist);
-            });
-        }
-
-        async Task ValidateCloudFormation(Func<AmazonCloudFormationClient, Task> execute)
-        {
-            var credentials = new BasicAWSCredentials(
-                Environment.GetEnvironmentVariable("AWS_Calamari_Access"),
-                Environment.GetEnvironmentVariable("AWS_Calamari_Secret"));
-            var config = new AmazonCloudFormationConfig { AllowAutoRedirect = true, RegionEndpoint = RegionEndpoint.APSoutheast1 };
-            using (var client = new AmazonCloudFormationClient(credentials, config))
-            {
-                await execute(client);
-            }
-        }
-
-        void DeployTemplate(string resourceName, string template)
-        {
-            var variablesFile = Path.GetTempFileName();
-            var variables = new CalamariVariables();
-            variables.Set("Octopus.Action.AwsAccount.Variable", "AWSAccount");
-            variables.Set("AWSAccount.AccessKey", Environment.GetEnvironmentVariable("AWS_Calamari_Access"));
-            variables.Set("AWSAccount.SecretKey", Environment.GetEnvironmentVariable("AWS_Calamari_Secret"));
-            variables.Set("Octopus.Action.Aws.Region", RegionEndpoint.APSoutheast1.SystemName);
-            variables.Save(variablesFile);
-
+            var template = CloudFormationFixtureHelpers.GetBasicS3Template(StackName);
```

Comment:
We could use the same helper methods to create the template as used in the above test here